tox.ini

```diff
@@ -12,16 +10,14 @@ setenv =
 commands = python runtests.py
 deps =
     django42: Django>=4.2,<5.0
-    django50: Django>=5.0,<5.1
     django51: Django>=5.1,<5.2
-    django52: Django>=5.2rc1,<6.0
+    django52: Django>=5.2,<5.3
```

Comment:
There will be no `5.3` version so, `<6.0` was a correct condition.

Reply:
@janbrasna Could you tweak that one back to `<6.0` please?